core/queryparser/sparql/src/main/java/org/eclipse/rdf4j/query/parser/sparql/ast/JJTSyntaxTreeBuilderState.java

```diff
@@ -5,10 +5,8 @@
  * are made available under the terms of the Eclipse Distribution License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/org/documents/edl-v10.php.
- *
- * SPDX-License-Identifier: BSD-3-Clause
- *******************************************************************************/
-/* Generated By:JavaCC: Do not edit this line. JJTSyntaxTreeBuilderState.java Version 7.0.11 */
+ ******************************************************************************/
+/* Generated By:JavaCC: Do not edit this line. JJTSyntaxTreeBuilderState.java Version 5.0 */
```

Comment:
Looks like you used an older version of JavaCC than the one we most recently upgraded to - worth rerunning with the newer version?

Reply:
Great, I will rerun it with a newer version.